### ISSUE TITLE
Learning rate scheduler

### ISSUE THREAD

**Issue by pabloruizponce** [pabloruizp/YOLOv1#10]
Understand and use lr scheduler to increase the lr used while training

**Comment by pabloruizponce** [comment]
_Our learning rate schedule is as follows: For the first epochs we slowly raise the learning rate from 10−3 to 10−2. If we start at a high learning rate our model often diverges due to unstable gradients. We continue training with 10−2 for 75 epochs, then 10−3 for 30 epochs, and finally 10−4 for 30 epochs._

**Comment by pabloruizponce** [comment]
lr_scheduler not working on change of scheduler

![Image](https://user-images.githubusercontent.com/46010361/180835106-8d0a0b42-99fa-49ef-a69d-3650f0ae7ea9.png)
